tests/storage/cdi_upload/conftest.py

```
@@ -11,8 +11,8 @@
 
 from libs.net.ip import random_ipv4_address
 from libs.net.udn import create_udn_namespace
-from utilities.constants import Images
-from utilities.constants.timeouts import TIMEOUT_1MIN, TIMEOUT_2MIN
+from utilities.constants import TIMEOUT_2MIN, TIMEOUT_40MIN, Images
+from utilities.constants.timeouts import TIMEOUT_2MIN
```

Flake8 (7.3.0)

[error] redefinition of unused 'TIMEOUT_2MIN' from line 14

(F811)